plugins/Makefile

```diff
@@ -1,4 +1,4 @@
 all: example_plugin
 
 example_plugin: example_plugin.c
-	gcc -fPIC -shared -I/usr/include/glib-2.0 -I/lib64/glib-2.0/include -Wcast-align -Wno-uninitialized -Wredundant-decls -Wwrite-strings -Wformat-nonliteral -Wmissing-format-attribute -Wsign-compare -Wtype-limits -Wuninitialized -Wall -Werror=implicit-function-declaration -Wl,--as-needed -Wmissing-prototypes -Waggregate-return -Wshadow -o example_plugin.so example_plugin.c
+	gcc -fPIC -shared $(shell pkgconf --cflags --libs libdnf) -Wcast-align -Wno-uninitialized -Wredundant-decls -Wwrite-strings -Wformat-nonliteral -Wmissing-format-attribute -Wsign-compare -Wtype-limits -Wuninitialized -Wall -Werror=implicit-function-declaration -Wl,--as-needed -Wmissing-prototypes -Waggregate-return -Wshadow -o example_plugin.so example_plugin.c
```

Comment:
Please use `pkg-config`, as not all consumers of the DNF stack are using `pkgconf` yet by default.

Reply:
Updated to use `pkg-config`, rebased on master -> 9db8dd954c7584a9c315d86383d4edc50e2571c8.